### ISSUE TITLE
Support Intern

### ISSUE THREAD

**Issue by mbertelsen** [wallabyjs/public#489]
Support the Intern testing framework. [The Inertn](https://theintern.github.io)

**Comment by Boolos** [comment]
Any plans on supporting this in the immediate future?

**Comment by ArtemGovorov** [comment]
@Boolos No, there're no plans for implementing the support in the immediate future.
